Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -6181,6 +6182,32 @@ def com_google_fonts_check_colorfont_tables(ttFont):
     else:
         yield PASS, "Looks good!"
 
+@check(
+    id = "com.google.fonts/check/space_on_gid1_for_colrv0",
+    rationale = """
+        A shaping bug in Windows 10 inserts whichever glyph is on GID 1
+        on top of color glyphs, spaces, and also seemingly at random,
+        for COLRv0 fonts.
+
+        Having the space glyph on GID 1 is a practical workaround for that.
+
+        See https://github.com/googlefonts/gftools/issues/609
+    """,
+    proposal = 'https://github.com/googlefonts/fontbakery/issues/3886'
+)
+def com_google_fonts_check_space_on_gid1_for_colrv0(ttFont):
+    """Put space glyph on GID 1 right after the .notdef glyph for COLRv0 fonts."""
+    SUGGESTED_FIX = ("To fix this, please reorder the glyph order so that"
+                     " the space glyph is on GID 1 right after the .notdef glyph.")
+    if 'COLR' in ttFont.keys() and ttFont['glyf'].glyphOrder[1] != 'space':
+        yield FAIL,\
+              Message('wrong-glyphorder',
+                      "This is a COLR font. As a workaround for a shaping bug in "
```

Comment:
it's not about shaping

```suggestion
                      "This is a COLR font. As a workaround for a rendering bug in "
```